browser/ui/tabs/features.cc

```diff
@@ -0,0 +1,21 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/tabs/features.h"
+
+#include "base/feature_list.h"
+
+namespace tabs {
+namespace features {
+
+const base::Feature kBraveVerticalTabs{"BraveVerticalTabs",
+                                       base::FEATURE_DISABLED_BY_DEFAULT};
+
+bool IsVerticalTabsEnabled() {
```

Comment:
personally I'm not a fan of these helper methods if they just check the feature value. It's easier to find things if you can just search for the flag name